api_tests/base/test_filters.py

```diff
@@ -222,3 +229,43 @@ def test_convert_value_float(self):
         field = FakeSerializer._declared_fields['float_field']
         value = self.view.convert_value(value, field)
         assert_equal(value, 42.0)
+
+
+
+class TestODMOrderingFilter(ApiTestCase):
+    class query:
+        title = ' '
+        def __init__(self, title):
+            self.title = title
+        def __str__(self):
+            return self.title
+
+    class queryn:
+        title = ' '
+        number = 0
+        def __init__(self, title, number):
+            self.title = title
+            self.number = number
+        def __str__(self):
+            return self.title
+
+    def test_filter_queryset_fails(self):
+        query_s = [self.query(x) for x in 'NewProj Zip Proj Activity'.split()]
```

Comment:
I am not a big fan of vague variable names. Could you change `query_s`, `sort_q`, and `sortt` into things that are very clear as to their purpose? Same with anything else similar.